web/templates/player/index.html.eex

```diff
@@ -1,5 +1,28 @@
-<h2>Listing players</h2>
+<h2>Active players:</h2>
+<table class="table">
+  <thead>
+    <tr>
+      <th>Name</th>
+
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+<%= for player <- @active_players do %>
+    <tr>
+      <td><%= player.name %></td>
+
+      <td class="text-right">
+        <%= link "Edit", to: player_path(@conn, :edit, player), class: "btn btn-default btn-xs" %>
+        <%= link "Deactivate", to: player_path(@conn, :update, player, %{player: %{active: !player.active}}), method: :put, class: "btn btn-default btn-xs" %>
```

Comment:
This was a pain. If you know of a better way to call `update` with the updated player params, please let me know